tests/max_config/examplesMaxConfig.ini

```diff
@@ -0,0 +1,55 @@
+﻿[DAQmx]
```

Comment:
I think we need to update the Testing section of CONTRIBUTING.md to mention this import as well